packages/google_maps_flutter/lib/src/marker.dart

```diff
@@ -274,15 +274,39 @@ class Marker {
   }
 
   @override
-  bool operator ==(Object other) {
-    if (identical(this, other)) return true;
-    if (other.runtimeType != runtimeType) return false;
-    final Marker typedOther = other;
-    return markerId == typedOther.markerId;
-  }
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+      other is Marker &&
+          runtimeType == other.runtimeType &&
+          markerId == other.markerId &&
+          alpha == other.alpha &&
+          anchor == other.anchor &&
+          consumeTapEvents == other.consumeTapEvents &&
+          draggable == other.draggable &&
+          flat == other.flat &&
+          icon == other.icon &&
+          infoWindow == other.infoWindow &&
+          position == other.position &&
+          rotation == other.rotation &&
+          visible == other.visible &&
+          zIndex == other.zIndex &&
+          onTap == other.onTap;
 
   @override
-  int get hashCode => markerId.hashCode;
+  int get hashCode =>
```

Comment:
nit: [hashValues](https://master-api.flutter.dev/flutter/dart-ui/hashValues.html) is typically used in the Flutter codebase.

Reply:
What do you mean by this?

Reply:
e.g https://github.com/flutter/plugins/blob/80318be69aa81b6b3b089285a6d52cb588c8f98e/packages/google_maps_flutter/lib/src/location.dart#L49